fontforge/encoding.c

```diff
@@ -2031,10 +2031,14 @@ return( NULL );
 	base = 256;
     else if ( enc->char_cnt<=0x10000 )
 	base = 0x10000;
-    encoded = malloc(base*sizeof(int32));
-    memset(encoded,-1,base*sizeof(int32));
-    unencoded = malloc(sf->glyphcnt*sizeof(int32));
-    unmax = sf->glyphcnt;
+    if ( base>0 ) {
+	encoded = malloc(base*sizeof(int32));
```

Comment:
Try null initializing encoded and unencoded on line 2019, these will be garbage if base/glyphcnt <= 0 and there's reallocs below.

Reply:
That was it!